src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js

```diff
@@ -179,6 +188,7 @@ define([
          * @param media {Object} the id of the media to edit
          */
         editSingleMedia: function(media) {
+            var $preview;
```

Comment:
There should be only one var statement in each function, so move this one together with `$info`